extensions/extensions-helm/aissemble-policy-decision-point-chart/templates/deployment.yaml

```diff
@@ -24,6 +24,10 @@ spec:
         {{- toYaml . | nindent 12}}
       {{- end }}
       serviceAccountName: {{ .Values.deployment.serviceAccountName }}
+      {{- if .Values.deployment.imagePullSecrets }}
```

Comment:
A: Would be cleaner to use `{{- with ...` and `{{- toYaml .` like the securityContext section above.

Reply:
Sounds good, will update to with